<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFeedback</class>
 <widget class="QDialog" name="DlgFeedback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpMessage">
     <property name="title">
      <string>Message *</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="txtMessage">
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblLimit">
        <property name="text">
         <string>0/2048</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="cmbFeelings">
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Positive</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../svnroot/newsflash/gui/resource.qrc">
           <normaloff>:/resource/16x16_ico_png/ico_happy.png</normaloff>:/resource/16x16_ico_png/ico_happy.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Neutral</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../svnroot/newsflash/gui/resource.qrc">
           <normaloff>:/resource/16x16_ico_png/ico_neutral.png</normaloff>:/resource/16x16_ico_png/ico_neutral.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Negative</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../svnroot/newsflash/gui/resource.qrc">
           <normaloff>:/resource/16x16_ico_png/ico_unhappy.png</normaloff>:/resource/16x16_ico_png/ico_unhappy.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>* Denotes a required field</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QLineEdit" name="editName"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblEmail">
          <property name="text">
           <string>Email *</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="editEmail"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblCountry">
          <property name="text">
           <string>Country</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="editVersion">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>The current software version</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="editPlatform">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>The platform, e.g. Windows XP, Windows 7, Ubuntu 9.10</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="lblPlatform">
          <property name="text">
           <string>Platform *</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name *</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lblVersion">
          <property name="text">
           <string>Version *</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="editCountry"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="grpAttachment">
     <property name="title">
      <string>Attachment (max ~2mb)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="editAttachment"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QProgressBar" name="progress">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editEmail</tabstop>
  <tabstop>editCountry</tabstop>
  <tabstop>editVersion</tabstop>
  <tabstop>editPlatform</tabstop>
  <tabstop>cmbFeelings</tabstop>
  <tabstop>txtMessage</tabstop>
  <tabstop>editAttachment</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>btnSend</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../svnroot/newsflash/gui/resource.qrc"/>
 </resources>
 <connections/>
</ui>
